app/src/components/market/common/history_section/history_select.tsx

```diff
@@ -336,6 +305,7 @@ export const History_select: React.FC<Props> = ({
       ) : (
         <HistoryChart
           data={data}
+          hasEnoughData={notEnoughData}
```

Comment:
Should be notEnoughData={notEnoughData}

Reply:
yeah totally messed up naming on that piece of data !